### ISSUE TITLE
vmagent crash with error "invalid memory address or nil pointer dereference"

### ISSUE THREAD

**Issue by ekapratama93** [VictoriaMetrics/VictoriaMetrics#1107]
vmagent crash when upgrading to v1.55.0

```
2021-03-03T06:21:34.507Z        info    VictoriaMetrics/lib/logger/flag.go:12   build version: vmagent-20210302-195139-tags-v1.55.0-0-g1252ca44d
2021-03-03T06:21:34.511Z        info    VictoriaMetrics/lib/logger/flag.go:13   command line flags
2021-03-03T06:21:34.512Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "csvTrimTimestamp" = "1ms"
2021-03-03T06:21:34.513Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "dryRun" = "false"
2021-03-03T06:21:34.513Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "enableTCP6" = "false"
2021-03-03T06:21:34.514Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "envflag.enable" = "false"
2021-03-03T06:21:34.514Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "envflag.prefix" = ""
2021-03-03T06:21:34.514Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "fs.disableMmap" = "false"
2021-03-03T06:21:34.515Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "graphiteListenAddr" = ""
2021-03-03T06:21:34.515Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "graphiteTrimTimestamp" = "1s"
2021-03-03T06:21:34.515Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "http.connTimeout" = "2m0s"
2021-03-03T06:21:34.516Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "http.disableResponseCompression" = "false"
2021-03-03T06:21:34.516Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "http.idleConnTimeout" = "1m0s"
2021-03-03T06:21:34.516Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "http.maxGracefulShutdownDuration" = "7s"
2021-03-03T06:21:34.517Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "http.pathPrefix" = ""
2021-03-03T06:21:34.517Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "http.shutdownDelay" = "0s"
2021-03-03T06:21:34.518Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "httpAuth.password" = "secret"
2021-03-03T06:21:34.518Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "httpAuth.username" = ""
2021-03-03T06:21:34.518Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "httpListenAddr" = ":8429"
2021-03-03T06:21:34.519Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "import.maxLineLen" = "104857600"
2021-03-03T06:21:34.519Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "influx.maxLineSize" = "262144"
2021-03-03T06:21:34.519Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "influxListenAddr" = ""
2021-03-03T06:21:34.520Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "influxMeasurementFieldSeparator" = "_"
2021-03-03T06:21:34.520Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "influxSkipMeasurement" = "false"
2021-03-03T06:21:34.520Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "influxSkipSingleField" = "false"
2021-03-03T06:21:34.521Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "influxTrimTimestamp" = "1ms"
2021-03-03T06:21:34.521Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "insert.maxQueueDuration" = "1m0s"
2021-03-03T06:21:34.521Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "loggerDisableTimestamps" = "false"
2021-03-03T06:21:34.522Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "loggerErrorsPerSecondLimit" = "0"
2021-03-03T06:21:34.522Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "loggerFormat" = "default"
2021-03-03T06:21:34.522Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "loggerLevel" = "INFO"
2021-03-03T06:21:34.523Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "loggerOutput" = "stderr"
2021-03-03T06:21:34.523Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "loggerTimezone" = "UTC"
2021-03-03T06:21:34.523Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "loggerWarnsPerSecondLimit" = "0"
2021-03-03T06:21:34.524Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "maxConcurrentInserts" = "16"
2021-03-03T06:21:34.524Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "maxInsertRequestSize" = "33554432"
2021-03-03T06:21:34.524Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "memory.allowedBytes" = "2GB"
2021-03-03T06:21:34.525Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "memory.allowedPercent" = "60"
2021-03-03T06:21:34.525Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "metricsAuthKey" = "secret"
2021-03-03T06:21:34.525Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "opentsdbHTTPListenAddr" = ""
2021-03-03T06:21:34.526Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "opentsdbListenAddr" = ""
2021-03-03T06:21:34.526Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "opentsdbTrimTimestamp" = "1s"
2021-03-03T06:21:34.526Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "opentsdbhttp.maxInsertRequestSize" = "33554432"
2021-03-03T06:21:34.527Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "opentsdbhttpTrimTimestamp" = "1ms"
2021-03-03T06:21:34.527Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "pprofAuthKey" = "secret"
2021-03-03T06:21:34.527Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.cluster.memberNum" = "0"
2021-03-03T06:21:34.528Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.cluster.membersCount" = "0"
2021-03-03T06:21:34.528Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.config" = "/etc/prometheus/prometheus.yml"
2021-03-03T06:21:34.528Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.config.dryRun" = "false"
2021-03-03T06:21:34.529Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.config.strictParse" = "false"
2021-03-03T06:21:34.529Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.configCheckInterval" = "5m0s"
2021-03-03T06:21:34.529Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.consulSDCheckInterval" = "30s"
2021-03-03T06:21:34.530Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.disableCompression" = "false"
2021-03-03T06:21:34.530Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.disableKeepAlive" = "false"
2021-03-03T06:21:34.530Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.discovery.concurrency" = "100"
2021-03-03T06:21:34.531Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.discovery.concurrentWaitTime" = "1m0s"
2021-03-03T06:21:34.531Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.dnsSDCheckInterval" = "30s"
2021-03-03T06:21:34.531Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.dockerswarmSDCheckInterval" = "30s"
2021-03-03T06:21:34.532Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.dropOriginalLabels" = "true"
2021-03-03T06:21:34.532Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.ec2SDCheckInterval" = "1m0s"
2021-03-03T06:21:34.532Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.eurekaSDCheckInterval" = "30s"
2021-03-03T06:21:34.533Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.fileSDCheckInterval" = "30s"
2021-03-03T06:21:34.533Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.gceSDCheckInterval" = "1m0s"
2021-03-03T06:21:34.533Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.kubernetes.apiServerTimeout" = "30m0s"
2021-03-03T06:21:34.533Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.kubernetesSDCheckInterval" = "30s"
2021-03-03T06:21:34.534Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.maxDroppedTargets" = "1000"
2021-03-03T06:21:34.534Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.maxScrapeSize" = "16777216"
2021-03-03T06:21:34.534Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.openstackSDCheckInterval" = "30s"
2021-03-03T06:21:34.534Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.streamParse" = "false"
2021-03-03T06:21:34.534Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.suppressDuplicateScrapeTargetErrors" = "false"
2021-03-03T06:21:34.534Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "promscrape.suppressScrapeErrors" = "true"
2021-03-03T06:21:34.534Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.basicAuth.password" = "secret"
2021-03-03T06:21:34.534Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.basicAuth.username" = ""
2021-03-03T06:21:34.534Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.bearerToken" = "secret"
2021-03-03T06:21:34.534Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.flushInterval" = "1s"
2021-03-03T06:21:34.535Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.label" = ""
2021-03-03T06:21:34.535Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.maxBlockSize" = "8388608"
2021-03-03T06:21:34.535Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.maxDiskUsagePerURL" = "0"
2021-03-03T06:21:34.535Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.proxyURL" = ""
2021-03-03T06:21:34.535Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.queues" = "4"
2021-03-03T06:21:34.535Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.rateLimit" = ""
2021-03-03T06:21:34.535Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.relabelConfig" = ""
2021-03-03T06:21:34.535Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.roundDigits" = ""
2021-03-03T06:21:34.535Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.sendTimeout" = ""
2021-03-03T06:21:34.535Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.showURL" = "false"
2021-03-03T06:21:34.535Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.significantFigures" = ""
2021-03-03T06:21:34.535Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.tlsCAFile" = ""
2021-03-03T06:21:34.535Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.tlsCertFile" = ""
2021-03-03T06:21:34.536Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.tlsInsecureSkipVerify" = ""
2021-03-03T06:21:34.536Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.tlsKeyFile" = "secret"
2021-03-03T06:21:34.536Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.tlsServerName" = ""
2021-03-03T06:21:34.536Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.tmpDataPath" = "/vmagentdata"
2021-03-03T06:21:34.536Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.url" = "secret"
2021-03-03T06:21:34.536Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "remoteWrite.urlRelabelConfig" = ""
2021-03-03T06:21:34.536Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "tls" = "false"
2021-03-03T06:21:34.536Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "tlsCertFile" = ""
2021-03-03T06:21:34.536Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "tlsKeyFile" = "secret"
2021-03-03T06:21:34.536Z        info    VictoriaMetrics/lib/logger/flag.go:20   flag "version" = "false"
2021-03-03T06:21:34.536Z        info    VictoriaMetrics/app/vmagent/main.go:88  starting vmagent at ":8429"...
2021-03-03T06:21:34.536Z        info    VictoriaMetrics/lib/memory/memory.go:47 limiting caches to 2000000000 bytes, leaving 25336183808 bytes to the OS according to -memory.allowedBytes=2GB
2021-03-03T06:21:34.538Z        info    VictoriaMetrics/lib/persistentqueue/fastqueue.go:51     opened fast persistent queue at "/vmagentdata/persistent-queue/1_063D5D0DDE22E267" with maxInmemoryBlocks=200, it contains 1028658146 pending bytes
2021-03-03T06:21:34.538Z        info    VictoriaMetrics/app/vmagent/remotewrite/client.go:142   initialized client for -remoteWrite.url="1:secret-url"
2021-03-03T06:21:34.539Z        info    VictoriaMetrics/app/vmagent/main.go:111 started vmagent in 0.003 seconds
2021-03-03T06:21:34.539Z        info    VictoriaMetrics/lib/httpserver/httpserver.go:82 starting http server at http://:8429/
2021-03-03T06:21:34.539Z        info    VictoriaMetrics/lib/httpserver/httpserver.go:83 pprof handlers are exposed at http://:8429/debug/pprof/
2021-03-03T06:21:34.540Z        info    VictoriaMetrics/lib/promscrape/scraper.go:91    reading Prometheus configs from "/etc/prometheus/prometheus.yml"
2021-03-03T06:21:34.596Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s:8001/api/v1/nodes"
2021-03-03T06:21:34.637Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s:8001/api/v1/nodes"
2021-03-03T06:21:34.897Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s:8001/api/v1/pods"
2021-03-03T06:21:34.965Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s:8001/api/v1/services"
2021-03-03T06:21:35.171Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s:8001/api/v1/pods"
2021-03-03T06:21:35.233Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s:8001/api/v1/endpoints"
2021-03-03T06:21:35.280Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s:8001/api/v1/namespaces/linkerd/pods"
2021-03-03T06:21:35.484Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s:8001/api/v1/pods"
2021-03-03T06:21:35.680Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s:8001/api/v1/pods"
2021-03-03T06:21:35.701Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s-2:8001/api/v1/nodes"
2021-03-03T06:21:35.715Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s-2:8001/api/v1/nodes"
2021-03-03T06:21:35.938Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s-2:8001/api/v1/pods"
2021-03-03T06:21:35.969Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s-2:8001/api/v1/services"
2021-03-03T06:21:36.252Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s-2:8001/api/v1/pods"
2021-03-03T06:21:36.288Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s-2:8001/api/v1/endpoints"
2021-03-03T06:21:36.320Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s-2:8001/api/v1/namespaces/linkerd/pods"
2021-03-03T06:21:36.334Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s-endeus:8001/api/v1/nodes"
2021-03-03T06:21:36.350Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s-endeus:8001/api/v1/nodes"
2021-03-03T06:21:36.440Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s-endeus:8001/api/v1/pods"
2021-03-03T06:21:36.456Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s-endeus:8001/api/v1/services"
2021-03-03T06:21:36.539Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s-endeus:8001/api/v1/pods"
2021-03-03T06:21:36.561Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s-endeus:8001/api/v1/endpoints"
2021-03-03T06:21:36.583Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://kurgle-k8s-endeus:8001/api/v1/namespaces/linkerd/pods"
2021-03-03T06:21:36.641Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://stagle-k8s:8001/api/v1/nodes"
2021-03-03T06:21:36.660Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://stagle-k8s:8001/api/v1/nodes"
2021-03-03T06:21:36.959Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://stagle-k8s:8001/api/v1/pods"
2021-03-03T06:21:37.094Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://stagle-k8s:8001/api/v1/services"
2021-03-03T06:21:37.356Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://stagle-k8s:8001/api/v1/pods"
2021-03-03T06:21:37.413Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://stagle-k8s:8001/api/v1/endpoints"
2021-03-03T06:21:37.443Z        info    VictoriaMetrics/lib/promscrape/discovery/kubernetes/api.go:249  started watcher for "http://stagle-k8s:8001/api/v1/namespaces/linkerd-viz/pods"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0x8423a6]

goroutine 53 [running]:
github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape.(*ScrapeWork).LabelsString(0x0, 0x203000, 0x203000)
        github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape/scrapework.go:118 +0x26
github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape.(*ScrapeWork).key(0x0, 0xc000a02720, 0xc0029a6f00)
        github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape/scrapework.go:106 +0x45
github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape.(*scraperGroup).update(0xc00095f6e0, 0xc00158e000, 0x1be1, 0x1c00)
        github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape/scraper.go:306 +0x16e
github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape.(*scrapeConfig).run.func1(0xc00024e740)
        github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape/scraper.go:236 +0xe5
github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape.(*scrapeConfig).run(0xc0008c36c0)
        github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape/scraper.go:240 +0x153
github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape.(*scrapeConfigs).add.func1(0xc0008c3600, 0xc0008c36c0)
        github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape/scraper.go:192 +0x59
created by github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape.(*scrapeConfigs).add
        github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape/scraper.go:190 +0x1dd
```

**Comment by zbindenren** [comment]
Same here. Rolling back to 1.54.1.

**Comment by f41gh7** [comment]
investigating.

**Comment by valyala** [comment]
The bug must be fixed in the commit 7906316741eb7deed2d0d7fb90ba50505ba37de3 . @zbindenren , @ekapratama93 , could you build `vmagent` from this commit and verify it doesn't panics anymore? See [build instructions for vmagent](https://victoriametrics.github.io/vmagent.html#how-to-build-from-sources)

**Comment by umezawatakeshi** [comment]
7906316 seems to work fine.

**Comment by valyala** [comment]
@umezawatakeshi , thanks for the confirmation!

The bugfix has been included in [v1.55.1](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing the issue as fixed.
